src/acme/ui/CustomerListUI.java

```diff
@@ -0,0 +1,353 @@
+package acme.ui;
+
+import java.awt.Color;
+import java.awt.Component;
+import java.awt.Font;
+import java.awt.Panel;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.sql.ResultSet;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Set;
+import java.util.UUID;
+
+import javax.swing.DefaultCellEditor;
+import javax.swing.JButton;
+import javax.swing.JLabel;
+import javax.swing.JOptionPane;
```

Comment:
In Eclipse, if you press (alt+shift+o) it will automatically decide which imports are needed. (Just general cleanup)

Reply:
I took them out